modules/check_in/spec/request/v2/appointments_request_spec.rb

```diff
@@ -235,10 +245,72 @@
         end
 
         it 'returns error' do
-          VCR.use_cassette 'check_in/facilities/get_facilities_500' do
-            VCR.use_cassette 'check_in/appointments/get_appointments_200' do
-              VCR.use_cassette 'check_in/map/security_token_service_200' do
-                get "/check_in/v2/sessions/#{id}/appointments", params: { start: start_date, end: end_date }
+          VCR.use_cassette 'check_in/clinics/get_clinics_500' do
```

Comment:
do we need this cassette here?

Reply:
Removed it in [3b0ce1a](https://github.com/department-of-veterans-affairs/vets-api/pull/17642/commits/3b0ce1aae8965571b1f8677904383f8199a65367)